guides/common/modules/con_hammer-cli-overview.adoc

```diff
@@ -0,0 +1,26 @@
+[id="Hammer-CLI-Overview_{context}"]
+= Hammer CLI Overview
+
+Hammer is a command-line tool to configure and manage a {ProjectServer} through CLI commands or automation in shell scripts.
+Hammer also provides an interactive shell. 
+
+Benefits of using Hammer:
+
+* Shell scripts created for Hammer allow for basic task automation.
+* Output from Hammer commands can be redirected to other tools.
+* The `--debug` option can be used to test responses to API calls before applying them in a script, for example, `hammer --debug organization list`.
+* Changes in the API are automatically reflected in Hammer, while scripts using the API directly have to be updated manually.
```

Comment:
This is very feature-centric and should be rewritten for user focus. Let me know if you need help with the phrasing.

Reply:
@jafiala  Can you take another look at the assembly about administration tools? I rewrote it with your tips on enhancing user focus in mind.

This is the commit: https://github.com/theforeman/foreman-documentation/pull/2607/commits/3eac1ccbc65699033fe3f35aa4db4a3ae1bcee9c
Here is a link: https://theforeman-foreman-documentation-preview-pr-2607.surge.sh/nightly/Planning_for_Project/index-satellite.html#Tools-for-Administration-of-Satellite_planning

The part I'm most unsure of is the introduction to the list of benefits ("Use the <tool> for the following benefits"). Would you choose a different wording?